### PR TITLE
Added a view - UserLocation as well component MapView

### PR DESCRIPTION
**Pull request by TracyH-P**
Within the UserLocation a view/ page was created to illistrate an input box for address, as with a button to ask the user to allow location data. Once either the button or a full address is entered and the user hits search, the map below through google maps API's updates to reflect the address inputed at a scale of zoom 14, the base are of the map is set with coordinates of detroit.

index was added for the new page, side bar menu still needs to be added 
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
     <meta charset="UTF-8">
     <link rel='icon' href="/favicon.ico">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Travel Companion</title>
-  </head>
+
+   
+    <!--Tracys implmentation changes for Index.html-->
+    <!--This is to link Semantic UI for user location design-->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.5.0/dist/semantic.min.css">
```
Comment (mrago256) [inline index.html:12]: It looks like you are using a new CSS library here. We already have Bootstrap for CSS, is there any specific reason for this inclusion? I don't care which one we end up using, but we should only use 1 to keep a consistent look thourhgout the site.

File: src/views/UserLocation.vue [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            }
+        },
+        methods: {
+            LocatorButtonPressed() {
+                this.spinner = true;
+                if (navigator.geolocation) {
+                    navigator.geolocation.getCurrentPosition(
+                        position => {
+                            const latitude = position.coords.latitude;
+                            const longitude = position.coords.longitude;
```
Comment (mrago256) [inline src/views/UserLocation.vue:45]: Nice. We will need these for saving the location to the database

**Comment by mrago256** [comment]
This is looking really good so far! Something I noticed is that you have `.vs` files included in this commit. I'm not really sure if there's a way to remove these right now, but once this is mergerd we'll probably want to remove them.

File: index.html (hunk 1 above)
Comment (mrago256) [inline index.html:12]: It looks like you are using a new CSS library here. We already have Bootstrap for CSS, is there any specific reason for this inclusion? It doesn't matter which one we end up using, but we should only use 1 to keep a consistent look throughout the site.

**Review by CCooley33 — APPROVED** [review]
Reviewed code, everything looks good to my knowledge! Might request a method to output current location later when we are able to handle saving locations

File: index.html (hunk 1 above)
Comment (TracyH-P) [reply index.html:12]: I dont fully remeeber at this time, but I can review the code, I want to say this was only added for the sake of a singular button... if so I can go through and try to use the bootstrap library, thatll be something i got to dig into

**Comment by Sharefa1** [comment]
Review the code - It looks good to me, Approved!

**Comment by mrago256** [comment]
Approving for sake of time before sprint end

**Review by mrago256 — APPROVED** [review]

**Pull request merged**